include/envoy/runtime/runtime.h

```diff
@@ -37,6 +39,21 @@ class Snapshot {
 public:
   virtual ~Snapshot() {}
 
+  /**
+   * The raw data from a single snapshot key.
+   */
+  struct Entry {
+    /**
+     * The raw runtime data.
+     */
+    std::string string_value;
```

Comment:
We are not fully consistent here, but I think most structs still use `_` at the end of members, so I would probably change it back.

Reply:
👌  done